lib/logflare/partners.ex

```diff
@@ -77,15 +77,12 @@ defmodule Logflare.Partners do
   Fetches user by token for a given Partner
   """
   def get_user_by_token(%Partner{token: token}, user_token) do
-    query =
-      from(p in Partner,
-        join: u in assoc(p, :users),
-        where: p.token == ^token,
-        where: u.token == ^user_token,
-        select: u
-      )
-
-    Repo.one(query)
+    from(p in Partner,
+      join: u in assoc(p, :users),
+      where: p.token == ^token and u.token == ^user_token,
+      select: u
+    )
+    |> Repo.one()
```

Comment:
While I prefer the merged `:where` part I think I prefer having variable for `from` and then pass it to `Repo.one` instead of using pipe there.